### PR TITLE
[CI] speed up packaging with more parallel stages

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Every single platform will be treated as a stage, then there will require more resources but it will be faster OTOH.

## Why is it important?

Packaging takes about 1 hour, so let's see if this reduces the build time a bit more.

## Tests

48 workers in parallel took -> WIP

### CODE REVIEW

File: .ci/packaging.groovy [hunk 1]
```diff
@@ -86,6 +86,22 @@ pipeline {
                   'x-pack/winlogbeat'
                 )
               }
+              axis {
+                name 'PLATFORMS'
+                values (
+                  '+all',
```
Comment (kuisathaverat) [inline .ci/packaging.groovy:92]: the axis should be like "+all linux/amd64" otherwise the build will not be correct

**Review by cachedout — APPROVED** [review]

**Comment by v1v** [comment]
This won't work since the docker images generation are required and cannot be split. so I'll close this to avoid any kind of complex pipeline to handle those scenarios

**Pull request closed without merging**